### PR TITLE
Add reminder

### PR DESCRIPTION
**Pull request by EVogel1999**
Adds a submission reminder to the modal.

First Page:
Screen Shot 2020-04-14 at 2 26 53 PM

Second Page:
Screen Shot 2020-04-14 at 2 26 59 PM

### CODE REVIEW

**Comment by EVogel1999** [comment]
Screen Shot 2020-04-15 at 1 57 56 PM

File: src/app/onion/shared/submit/submit.component.html [hunk 1]
```diff
@@ -13,6 +17,27 @@
...
+              <li>All materials that are needed for this Learning Object are uploaded</li>
+              <li>All materials (i.e. PowerPoints, PDFs, videos etc.) are accessible under Section 508 of the Americans with Disability Act</li>
+              <li>All child objects of this Learning Object are in the hierarchy</li>
+            </ul>
+            <div class="license-selection">
+              <span tabindex="0" class="statement">I have checked this Learning Object for the errors listed above and would like to continue my submission to a collection</span>
+              <clark-checkbox aria-label="Accept that you have reviewed the submission reminder" (action)="submissionReminder = $event"></clark-checkbox>
+            </div>
+            <div class="submit-btn btn-group center">
+              <button class="button good" [ngClass]="{'disabled': !submissionReminder}" [disabled]="!submissionReminder" (activate)="advance()">Continue</button>
```
Comment (pzalep1) [inline src/app/onion/shared/submit/submit.component.html:35]: Aria-label

File: src/app/onion/shared/submit/submit.component.html [hunk 2]
```diff
@@ -13,6 +17,27 @@
...
+              <button class="button neutral" (activate)="closeModal()">Cancel</button>
```
Comment (pzalep1) [inline src/app/onion/shared/submit/submit.component.html:36]: Aria-label

**Comment by pzalep1** [comment]
Can we get new screenshots please?

File: src/app/onion/shared/submit/submit.component.html [hunk 3]
```diff
@@ -2,6 +2,10 @@
   <div class="submit" #popupInner>
     <div class="banner">
       <clark-carousel [action$]="carouselAction$">
+          <ng-template>
+            <div class="banner__title">Submission Reminders</div>
+            <div class="banner__sub">Please read and review the following.</div>
```
Comment (pzalep1) [inline src/app/onion/shared/submit/submit.component.html:7]:
```suggestion
            <div class="banner__sub">Please read and review before continuing</div>
```

**Comment by EVogel1999** [comment]
Screen Shot 2020-04-20 at 2 22 43 PM

File: src/app/onion/shared/submit/submit.component.html [hunk 4]
```diff
@@ -13,6 +17,29 @@
...
+        <div class="carousel__item">
+          <div class="center-content">
+            <p>Before continuing in the submission process, please verify that you have:</p>
+            <ul>
+              <li>Checked for spelling and grammar errors</li>
+              <li>All materials that are needed for this Learning Object are uploaded</li>
+              <li>All materials (i.e. PowerPoints, PDFs, videos etc.) are accessible under Section 508 of the Americans with Disability Act</li>
+              <li>All child objects of this Learning Object are in the hierarchy</li>
+              <li>All materials are correctly linked (i.e. YouTube videos or VM downloads)</li>
+              <li>All of the learning outcomes you would like to include are written</li>
```
Comment (pzalep1) [inline src/app/onion/shared/submit/submit.component.html:30]:
```suggestion
              <li>All of the learning outcomes you would like to include are added</li>
```

**Pull request merged**